modules/ai_engine_embedding/src/Service/EntityUpdate.php

```diff
@@ -241,7 +248,7 @@ public function upsertDocument(EntityInterface $entity) {
 
     if ($response === NULL) {
       $this->logger->notice(
-        'Unable to upsert node @id to vector database. Response not successful',
+        'Unable to upsert node @id to vector database. Response not successful.  Could the azure embedding service URL be incorrect?',
```

Comment:
Looks like an extra space here, and message could be more direct, such as: "Check the Azure embedding service URL."

Reply:
Yep, that's what I get for copying below notices.  hahaha. Let me fix that one.